openshift-ci/run_metallb_e2e.sh

```
@@ -1,6 +1,9 @@
 #!/usr/bin/bash
 set -euo pipefail
 
+metallb_dir="$(dirname $(readlink -f $0))"
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)